README.md

```diff
@@ -26,13 +26,153 @@ cd bidbot
 make install
 ```
 
-## Getting Started
-1. Make sure `$GOPATH/bin` is in your `$PATH`, i.e., `export PATH=$GOPATH/bin:$PATH`.
-1. `bidbot init`.
-1. The output will ask you to sign a token with the owner address of your miner.
-1. Configure your _ask price_, other bid settings, and auction filters. See `bidbot help daemon` for details. You can edit the configuration file generated by `bidbot init` or use the equivalent flag for any given option.
-1. Use the signature you generated above to start the daemon: `bidbot daemon --miner-addr [address] --wallet-addr-sig [signature]`
-1. Good luck! Your `bidbot` will automatically bid in open deal auctions. If it wins an auction, the broker will automatically start making a deal with the Lotus wallet address used in step 3.
+# What is the Storage Broker?
```

Comment:
Table of Contents should be updated accordingly

Reply:
Also maybe worth adding some background info on why storage broker is mentioned here, or maybe change line 3 above to mention storage broker.

Reply:
Agree, I'll add some extra words here.